defrag/__init__.py

```diff
@@ -15,10 +15,70 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 import logging
+from typing import Any, Dict, List
 from fastapi import FastAPI
-from pathlib import Path
+from pydantic import BaseSettings
+from functools import lru_cache
 import configparser
-import os
+
+
+class Settings(BaseSettings):
+    """
+    Automatically set by `python-dotenv` upon reading a ./.env file
+    (if any)
+    """
+    admin_emails: List[str] = [
+        "karatekhd@opensuse.org", "nycticorax@opensuse.org"]
+    app_name: str = "defrag-api"
+    app_port: str
+    app_host: int
+    bugzilla_user: str
+    bugzilla_password: str
+    mongo_pwd: str
+    mongo_name: str
```

Comment:
Correct, I had set it up so that we are ready to use mongo, which I am also running some tests on with `adrien/feature/mongodb`. Would you like to talk about mongodb first or do you prefer me removing these two line first?

Reply:
Just comment them our for now please ;)
For Mongo we probably should have a small meeting in the future